src/main/java/nl/mfarr/backendcontrolleruitwerkingen/exceptions/RecordNotFoundException.java

```diff
@@ -1,7 +1,11 @@
 package nl.mfarr.backendcontrolleruitwerkingen.exceptions;
 
-public class RecordNotFoundException {
+import nl.mfarr.backendcontrolleruitwerkingen.controllers.ExceptionController;
+
+public class RecordNotFoundException extends ExceptionController {
```

Comment:
Wanneer je een cutsom exception maakt, zoals de RecordNotFoundException, dan extend je deze van RuntimeException. RuntimeException is de klasse waar alle exception van extenden die je tijdens het runnen (tijdens runtime) tegen kunt komen. IndexOutOfBoundsExceptions is daar zo'n voorbeeld van, wanneer je `myList.get(100)` doet op een lijst waar maar 10 dingen in staan.

Reply:
We gaan dit nog wel nabespreken in de huiswerkles morgen, dus misschien dat je daar bij kunt zijn